Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -47,47 +51,53 @@ $ curl -X GET http://10.109.65.142:9080/metrics 2>/dev/null | grep csi
-liveness and grpc metrics into ceph monitoring.
+Check the [monitoring documentation](../Monitoring/ceph-monitoring.md) to see how to integrate CSI
+liveness and GRPC metrics into Ceph monitoring.
 
 ## Dynamically Expand Volume
 
-### Prerequisite
+### Prerequisites
 
 * For filesystem resize to be supported for your Kubernetes cluster, the
-  kubernetes version running in your cluster should be >= v1.15 and for block
-  volume resize support the Kubernetes version should be >= v1.16. Also,
-  `ExpandCSIVolumes` feature gate has to be enabled for the volume resize
+  Kubernetes version must be >= v1.15 and for block
+  volume resize support the Kubernetes version should be >= v1.16. Also, the
+  `ExpandCSIVolumes` feature gate must be enabled for the volume resize
   functionality to work.
 
 To expand the PVC the controlling StorageClass must have `allowVolumeExpansion`
 set to `true`. `csi.storage.k8s.io/controller-expand-secret-name` and
-`csi.storage.k8s.io/controller-expand-secret-namespace` values set in
-storageclass. Now expand the PVC by editing the PVC
+`csi.storage.k8s.io/controller-expand-secret-namespace` values set in the
+storageclass. Now expand the PVC by editing the PVC's 
 `pvc.spec.resource.requests.storage` to a higher values than the current size.
-Once PVC is expanded on backend and same is reflected size is reflected on
-application mountpoint, the status capacity `pvc.status.capacity.storage` of
-PVC will be updated to new size.
+Once the PVC is expanded on the back end and the new size is reflected on
+the application mountpoint, the status capacity `pvc.status.capacity.storage` of
+the PVC will be updated to the new size.
 
 ## RBD Mirroring
 
-To support RBD Mirroring, the [CSI-Addons sidecar](https://github.com/csi-addons/kubernetes-csi-addons#readme) will be started in the RBD provisioner pod.
-The CSI-Addons supports the VolumeReplication operation. The volume replication controller provides common and reusable APIs for storage disaster recovery. It is based on [csi-addons/spec](https://github.com/csi-addons/spec) specification and can be used by any storage provider.
-It follows the controller pattern and provides extended APIs for storage disaster recovery. The extended APIs are provided via Custom Resource Definitions (CRDs).
+To support RBD Mirroring,
+the [CSI-Addons sidecar](https://github.com/csi-addons/kubernetes-csi-addons#readme)
+will be started in the RBD provisioner pod. CSI-Addons support the `VolumeReplication`
+operation. The volume replication controller provides common and reusable APIs for
+storage disaster recovery. It is based on the [csi-addons/spec](https://github.com/csi-addons/spec)
+specification and can be used by any storage provider.
+It follows the controller pattern and provides extended APIs for storage disaster recovery.
+The extended APIs are provided via Custom Resource Definitions (CRDs).
 
 ### Prerequisites
```

Comment:
Let's remove this section, no need to mention the min version since Rook's is already higher than this.